core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -17,9 +17,7 @@ l.layout(permission:app.SYSTEM_READ, title:my.displayName, cssclass:request.getP
             l.icon(class: 'icon-secure icon-xlg')
             text(my.displayName)
         }
-        if (!h.hasPermission(app.ADMINISTER)) {
-            set("readOnlyMode", "true")
-        }
+        set("readOnlyMode", !h.hasPermission(app.ADMINISTER))
```

Comment:
Seems like an oversight to use `Functions` without need.

Reply:
are you saying `Functions` should be used here or h?

Reply:
`app.hasPermission` as in some other of these changes.

Reply:
done

Reply:
(Or `Jenkins.get()` which is the nicer, type-safer variant for Groovy views, but really the improvement is marginal. Not worth redoing, just a recommendation for the future.)